lib/versioning/pvp/index.ts

```diff
@@ -0,0 +1,80 @@
+import { regEx } from '../../util/regex';
+import { GenericVersion, GenericVersioningApi } from '../loose/generic';
+import { VersioningApi } from '../types';
+
+export const id = 'pvp';
+export const displayName = 'PVP';
+export const urls = ['https://pvp.haskell.org'];
+export const supportsRanges = false;
+
+/**
+ * At least 3 components and no leading 0s
```

Comment:
```suggestion
 * At least three components and no leading zeros
```

This should be easier to read. What do you think?